heron/spi/src/java/com/twitter/heron/spi/utils/SchedulerUtils.java

```diff
@@ -438,4 +438,20 @@ public static boolean curlAndExtractPackage(
 
     return true;
   }
+
+  /**
+   * This method finds the container with highest resource requirement and returns the resource.
+   * Currently only RAM is used for max identification.
+   */
+  public static PackingPlan.Resource getMaxRequiredResource(PackingPlan packingPlan) {
```

Comment:
This method is moved from AuroraScheduler, maybe we should put this refactor into a separate PR.